src/database-sqlite3.cpp

```diff
@@ -107,6 +160,12 @@ void Database_SQLite3::openDatabase()
 		throw FileNotGoodException("Cannot open database file");
 	}
 
+	if (sqlite3_busy_handler(m_database, sqlite3BusyHandler, busy_handler_data) != SQLITE_OK) {
```

Comment:
The advantage of not using SQLOK, is that the exception message can be customized to mention what operation was being performed...  I propose to modify SQLOK, adding a third parameter: an additional message to be included in the exception.

I created a separate patch that does this, and changed the code so the modified SQLOK is used where applicable.